README.md

```diff
@@ -104,50 +104,101 @@ Refer to the [CONTRIBUTING.md](./CONTRIBUTING.md) file for guidelines on contrib
 |ResNet-18|Filter pruning, 30%, geometric median criterion|ImageNet|69.76|68.97|
-|ResNet-34|None|ImageNet|-|73.31|
+|ResNet-34|None|ImageNet|73.31|-|
 |ResNet-34|Filter pruning, 30%, magnitude criterion|ImageNet|73.31|72.54|
-|ResNet-34|Filter pruning, 30%, geometric median criterion|ImageNet|73.31|72.60|
-|SSD300-BN|None|VOC12+07|-|78.28|
-|SSD300-BN|INT8|VOC12+07|78.28|78.07|
-|SSD300-BN|INT8 + Sparsity 70% (Magnitude)|VOC12+07|78.28|78.01|
-|SSD512-BN|None|VOC12+07|-|80.26|
-|SSD512-BN|INT8|VOC12+07|80.26|80.02|
-|SSD512-BN|INT8 + Sparsity 70% (Magnitude)|VOC12+07|80.26|79.98|
-|UNet|None|CamVid|-|71.95|
+|ResNet-34|Filter pruning, 30%, geometric median criterion|ImageNet|73.31|72.62|
+
+#### Object detection
+
+|Model|Compression algorithm|Dataset|PyTorch FP32 baseline|PyTorch compressed accuracy|
+| :---: | :---: | :---: | :---: | :---: |
+|SSD300-BN|None|VOC12+07|78.28|-|
+|SSD300-BN|INT8|VOC12+07|78.28|78.12|
+|SSD300-BN|INT8 + Sparsity 70% (Magnitude)|VOC12+07|78.28|77.94|
+|SSD512-BN|None|VOC12+07|80.26|-|
+|SSD512-BN|INT8|VOC12+07|80.26|80.09|
+|SSD512-BN|INT8 + Sparsity 70% (Magnitude)|VOC12+07|80.26|79.88|
+
+#### Semantic segmentation
+
+|Model|Compression algorithm|Dataset|PyTorch FP32 baseline|PyTorch compressed accuracy|
+| :---: | :---: | :---: | :---: | :---: |
+|UNet|None|CamVid|71.95|-|
 |UNet|INT8|CamVid|71.95|71.66|
 |UNet|INT8 + Sparsity 60% (Magnitude)|CamVid|71.95|71.72|
-|ICNet|None|CamVid|-|67.89|
+|ICNet|None|CamVid|67.89|-|
 |ICNet|INT8|CamVid|67.89|67.87|
 |ICNet|INT8 + Sparsity 60% (Magnitude)|CamVid|67.89|67.24|
-|UNet|None|Mapillary|-|56.23|
+|UNet|None|Mapillary|56.23|-|
 |UNet|INT8|Mapillary|56.23|56.12|
 |UNet|INT8 + Sparsity 60% (Magnitude)|Mapillary|56.23|56.0|
 
 
+#### NLP
+
+|Model|Compression algorithm|Dataset|PyTorch FP32 baseline|PyTorch compressed accuracy|
+| :---: | :---: | :---: | :---: | :---: |
+|BERT-base-chinese|None|XNLI|77.68|-|
+|BERT-base-chinese|INT8|XNLI|77.68|77.22|
+|BERT-large (Whole Word Masking)|None|SQuAD v1.1|93.21 (F1)|-|
+|BERT-large (Whole Word Masking)|INT8|SQuAD v1.1|93.21 (F1)|92.68 (F1)|
+|RoBERTa-large|None|MNLI|90.6 (matched)|-|
+|RoBERTa-large|None|MNLI|90.6 (matched)|89.25 (matched)|
+|DistilBERT-base|None|SST-2|91.1|-|
+|DistilBERT-base|INT8|SST-2|91.1|90.3|
+|MobileBERT|None|SQuAD v1.1|89.98 (F1)|-|
+|MobileBERT|INT8|SQuAD v1.1|89.98 (F1)|89.3 (F1)|
+
+
+#### Object detection (3rd party)
+
+|Model|Compression algorithm|Dataset|PyTorch FP32 baseline|PyTorch compressed accuracy|
+| :---: | :---: | :---: | :---: | :---: |
+|RetinaNet-ResNet50-FPN|None|COCO2017|35.6 (avg box mAP)|-|
+|RetinaNet-ResNet50-FPN|INT8|COCO2017|35.6 (avg box mAP)|35.3 (avg box mAP)|
+|RetinaNet-ResNet50-FPN|INT8 + Sparsity 50%|COCO2017|35.6 (avg box mAP)|34.7 (avg box mAP)|
+|RetinaNet-ResNeXt101-64x4d-FPN|None|COCO2017|39.6 (avg box mAP)|-|
+|RetinaNet-ResNeXt101-64x4d-FPN|INT8|COCO2017|39.6 (avg box mAP)|39.1 (avg box mAP)|
+
+
```

Comment:
You could also add Instance Segmentation results:

|Model|Compression algorithm|PyTorch FP32 baseline|PyTorch compressed accuracy|
| ------------- |:-------------:|:-------------:|:-------------:|
|MaskRCNN-ResNet50-FPN|INT8|0.379 (AP@IoU=0.50:0.95)|0.372 (AP@IoU=0.50:0.95) |

Reply:
We'd better provide a configuration file to reproduce it then; in the near future, I think, we will be validating third-party integration checkpoints along with the regular ones.

Reply:
These results were obtained by integrating NNCF with torchvision. Hence we'll need to provide a whole separate torchvision patch to validate them. Given that torchvision is not a widely used package for OD models, it might be not reasonable. Alternatively, I can add an mmdetction config for the same Mask-RCNN model and update the results. @AlexKoff88 what do you think?

Reply:
We need to provide a config to claim these numbers. As far as I remember in order to reproduce the results patching of torchvision code is required which means that we need to provide and support this patch. Personally, I don't like this idea.

Reply:
I have a different comment. Why do we need rows with "None" compression type while we provide baseline metrics at the same time?

Reply:
I initially reproduced Mask-RCNN quantization within 1% mAP drop in mmdetection. Can provide the corresponding config in the future, if needed.

Reply:
@AlexKoff88 a good point there. Will remove in a follow-up PR.